src/set_theory/game/nim.lean

```diff
@@ -294,73 +298,35 @@ lemma grundy_value_eq_mex_right : ∀ (G : pgame) [G.impartial],
   apply grundy_value_neg
 end
 
+/-- The Grundy value of the sum of two nim games with natural numbers of piles equals their bitwise
+xor. -/
 @[simp] lemma grundy_value_nim_add_nim (n m : ℕ) :
```

Comment:
Would you mind restoring the level of human readable comments that were here previously? Obviously the proof has changed a bit, so not the exact same ones, just a similar density. :-)

Reply:
Comments have been restored.